third-party/cmake/LibvtermCMakeLists.txt

```diff
@@ -18,7 +18,7 @@ foreach(file ${TBL_FILES})
   set(tname encoding/${basename}.inc)
   add_custom_command(OUTPUT
     COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_BINARY_DIR}/encoding/
-    COMMAND ${PERL_EXECUTABLE} -CSD ${CMAKE_SOURCE_DIR}/tbl2inc_c.pl ${file} > ${CMAKE_BINARY_DIR}/${tname}
+    COMMAND ${CMAKE_COMMAND} -P ${CMAKE_SOURCE_DIR}/tbl2inc_c.cmake ${file} ${CMAKE_BINARY_DIR}/${tname}
```

Comment:
If it's not much trouble, would be nice in the `perl_FOUND` case to compare the output of `tbl2inc_c.pl` to that of `tbl2inc_c.cmake`. As a sanity check. Then we know there's no bugs in `tbl2inc_c.cmake`. (Amazing effort, by the way!)